Formula/m/morpheus.rb

```diff
@@ -0,0 +1,98 @@
+class Morpheus < Formula
+  desc "Modelling environment for multi-cellular systems biology"
+  homepage "https://morpheus.gitlab.io/"
+  url "https://gitlab.com/morpheus.lab/morpheus/-/archive/v2.3.7/morpheus-v2.3.7.tar.gz"
+  sha256 "ad5694a098e4752db53659ee983c3ae417a43747320e73c3005f6cf88b52d55c"
+  license "BSD-3-Clause"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+(?:_?\d+)?)$/i)
+  end
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "ninja" => :build
+  depends_on "ffmpeg"
+  depends_on "graphviz"
+  depends_on "libomp"
+  depends_on "libtiff"
+  depends_on "qt@5"
+
+  uses_from_macos "bzip2"
+  uses_from_macos "libxml2"
+  uses_from_macos "zlib"
+
+  def install
+    # has to build with Ninja until: https://gitlab.kitware.com/cmake/cmake/-/issues/25142
+    args = []
+    args << "-G Ninja"
+
+    if OS.mac?
+      args << "-DMORPHEUS_RELEASE_BUNDLE=ON"
+      args << "-DMORPHEUS_SBML=OFF" # SBML import currently disabled due to libSBML build errors with some macOS SDKs
+    end
+
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+
+    if OS.mac?
+      bin.write_exec_script "#{prefix}/Morpheus.app/Contents/MacOS/morpheus"
+      bin.write_exec_script "#{prefix}/Morpheus.app/Contents/MacOS/morpheus-gui"
+    end
+  end
+
+  def post_install
+    if OS.mac? && File.read("#{prefix}/Morpheus.app/Contents/Info.plist").include?("HOMEBREW_BIN_PATH")
+      # Set PATH environment variable including Homebrew prefix in macOS app bundle
+      inreplace "#{prefix}/Morpheus.app/Contents/Info.plist", "HOMEBREW_BIN_PATH", "#{HOMEBREW_PREFIX}/bin"
+    end
+  end
```

Comment:
You don't need this in `post_install`; you can move this to the end of the `install` method.

Reply:
The suggestion would potentially break the software as it wouldn't be aware of the target system's `"#{HOMEBREW_PREFIX}/bin"` path, particularly if shipped via bottles, and therefore I would respectfully reject this one.

Reply:
Why wouldn't it be aware of that?

Reply:
Nope, it won't break. `brew` knows to adjust `$HOMEBREW_PREFIX` references in text files when pouring bottles.

Reply:
Apologies for having to ask again. 😅

As I understand it, the `inreplace` operation in the `install` block is not executed when a bottle is poured.

The goal here is to communicate non-standard Homebrew prefixes, that may have been configured by the user of the target system, to the application bundle (for this purpose, upstream provides the placeholder string `HOMEBREW_BIN_PATH` in the Info.plist) in order to make runtime dependencies installed via Homebrew findable in the application context as well (see the [Info.plist reference](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/LaunchServicesKeys.html#//apple_ref/doc/uid/20001431-106825)).

So it would make sense to do this replacement not in the bottling step in the `install` block on the CI system, but after the bottle has been poured during `post_install` on the user's target system.

Unless I'm missing something here?

Reply:
You are.

As I've explained: `brew` knows to adjust references to `$HOMEBREW_PREFIX` inside text files on the build machine so that they are appropriate for the machine that the bottle is being poured in (which may have a different `HOMEBREW_PREFIX`).

You don't need to manually manage this inside a `post_install` block because `brew` already knows to fix it up for you.